module-2/backend/week_one.md

```diff
@@ -5,30 +5,57 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - retrive web page, POST - post info to a web page, DELETE - delete a record, UPDATE - update a record, PUT - update or create a record

```

Comment:
less specifically about a web page and getting information from it and more about data in general...we wont always be displaying the data we want to get